tornjak-frontend/src/components/tornjak-helper.js

```diff
@@ -119,19 +110,29 @@ class TornjakHelper extends Component {
 
   // workloadEntry takes in an entry metadata and workload attestor info for specified agents
   // returns entry metadata info for dashboard table
-  workloadEntry(entry, workLoadAttestorInfo) {
+  workloadEntry(entry, workLoadAttestorInfo, globalAgents, globalEntries) {
     var thisSpiffeId = this.SpiffeHelper.getEntrySpiffeid(entry)
     var thisParentId = this.SpiffeHelper.getEntryParentid(entry)
+    var canonicalParentId = thisParentId
     // get tornjak metadata
-    var metadata_entry = this.SpiffeHelper.getAgentMetadata(thisParentId, workLoadAttestorInfo);
+    var parentAgent = globalAgents.find(a => this.SpiffeHelper.getAgentSpiffeid(a) === thisParentId)
+    if (parentAgent === undefined) {
```

Comment:
alternative:
```
if (!parentAgent) {
```

Reply:
File not of concern on this PR. Will be addressed in the next consecutive PR.